doc/ug/dv_methodology.md

```diff
@@ -131,7 +131,7 @@ The actual UVM testbenches for the Comportable IPs extend from this library as t
 In addition to the above library of classes, there are several common plug-and-play verification compoments (a.k.a. universal verification components or UVCs) provided for use in testbenches at `hw/dv/sv` location.
 A few examples of these are as follows:
 
-*  [Common interfaces]({{< relref "hw/dv/sv/common_ifs/README" >}})
```

Comment:
I see that some references were mentioning `README` and not `README.md`. Not sure if it is intentional.

Reply:
My preference would be to leave off `.md` extensions, as then a page can change from a single page to a bundle without rewriting links anywhere.  For `README.md` that's not going to happen, so it can be left with a `.md`.